src/utils.ts

```diff
@@ -97,29 +97,111 @@ export function sleep(ms: number): Promise<void> {
   return new Promise((resolve) => setTimeout(resolve, ms));
 }
 
+export const downloadAxios = axios.create({
+  responseType: "stream",
+  raxConfig: {
+    retry: 5, // number of retry when facing 400 or 500
+    onRetryAttempt(err) {
+      const cfg = rax.getConfig(err);
+      if (err.response?.status === 412) {
+      }
```

Comment:
Any logics in here?
[Here](https://github.com/planetarium/9c-launcher/pull/989/files#diff-39b2554fd18da165b59a6351b1aafff3714e2a80c1435f2de9706355b4d32351R116-R122) is the logics you intended I guess.